src/test/java/edu/ucsb/cs156/happiercows/services/SystemInfoServiceImplTests.java

```diff
@@ -27,7 +27,7 @@ class SystemInfoServiceImplTests  {
   @Test
   void test_getSystemInfo() {
     SystemInfo si = systemInfoService.getSystemInfo();
-    assertTrue(si.getSpringH2ConsoleEnabled());
+    assertTrue(!si.getSpringH2ConsoleEnabled());
```

Comment:
```suggestion
    assertFalse(si.getSpringH2ConsoleEnabled());
```

This is the preferred style